base/abstractarray.jl

```diff
@@ -1073,6 +1073,11 @@ function copy(a::AbstractArray)
     copymutable(a)
 end
 
+function copy(a::Core.ImmutableArray)
+    @_propagate_inbounds_meta
+    deepcopy(a)
```

Comment:
`deepcopy` is not a legal implementation of `copy` (or vice versa)

Reply:
Fixed this, though thinking more about it, the idea of a shallow copy of an immutable array is a bit strange in general. Maybe the shallow copy should be updated if someone were to `thaw` and then modify the original array?